keg_login/forms.py

```diff
@@ -90,6 +90,15 @@ class ResetPasswordForm(Form):
     return ResetPasswordForm
 
 
+def make_lock_form():
```

Comment:
That seems to be the pattern for the rest of the forms in keg-login. I think this was just done for consistency.